config/jobs/cert-manager/cert-manager/cert-manager-periodics.yaml

```diff
@@ -13,6 +17,7 @@ periodics:
     labels:
       preset-make-volumes: "true"
       preset-service-account: "true"
+    max_concurrency: 8
```

Comment:
note: adding `max_concurrency` everywhere is a recurring change. This was specified in some places and not others. The values are arbitrary but it feels like we should have this specified everywhere.